app/models/medical_history.rb

```diff
@@ -26,4 +26,11 @@ class MedicalHistory < ApplicationRecord
   enum receiving_treatment_for_hypertension: MEDICAL_HISTORY_ANSWERS, _prefix: true
   enum diabetes: MEDICAL_HISTORY_ANSWERS, _prefix: true
   enum diagnosed_with_hypertension: MEDICAL_HISTORY_ANSWERS, _prefix: true
+
+  def risk_history?
```

Comment:
This name confused me at first. What we're really describing is "does this patient have a history that make them at risk?"

Maybe `indicates_risk?` is better? What do you think?

Reply:
Yeah. That works. Had trouble naming this quite a bit, since `history` is already contextual.